README.md

```diff
@@ -1 +1,55 @@
-# orm-mapping-db-to-ruby-object-lab
+# ORM Mapping Database Rows to Ruby Objects
+
+## Objectives
+
+1. Write a method that turns database rows into Ruby objects 
+2. Write methods that allow you to search a database
+
+## Overview
+
+We are going to continue building out the `Student` class of our app. We are in the middle of building an app to help a public school's adminstrators keep track of their students. In this app we have a `Student` class. Each individual students has two attributes, a `name` and a `grade`, i.e. 9th, 10th, 11th, etc. 
+
+Our administrators need to save the student objects that this class produces to a database. We've set up the database for you in the `config/environment.rb` file. 
+
+Let's take a look:
+
+```ruby
+require 'sqlite3'
+require_relative '../lib/student'
+
+DB = {:conn => SQLite3::Database.new("db/students.db")}
+```
+Your connection to the database can be referred to, throughout your program, like this: `DB[:conn]`. 
+
+So far we built the ability to create the students table in the database (`Student.create_table`), drop that table (`Student.drop_table`), and `#save` a student to the database.
+
+Now, we will need to create a method that takes a row from the database and turns it back into a Student object. We will call this `#new_from_db`.
```

Comment:
after much research, class methods should start with a dot so (`.new_from_db`) and instance methods start with a pound so (`#new_from_db`). Pretty sure this is a class method